src/kubernetes-ingress-controller/references/custom-resources.md

```diff
@@ -251,6 +252,24 @@ meaning it will be executed for every request that is proxied via Kong.
 > available, it is the preferred means of configuring that setting, and the
 > annotation value will take precedence over a KongIngress value if both set
 > the same setting.
+{% endif_version -%}
+{% if_version gte:2.8.x -%}
+{:.note}
+> As of version 2.8, KongIngress sections other than `upstream` are
```

Comment:
You can copy/paste from my revision suggestions on the previous page.

Reply:
It looks like these were all Service->service and left as-is as such, so no change here either.